source/beamline/BioXAS/BioXASSideTableCoordinator.cpp

```diff
@@ -388,8 +371,11 @@ void BioXASSideTableCoordinator::manipulateVerticalPVChange(double upstreamHeigh
 
 void BioXASSideTableCoordinator::manipulateHorizontalPVChange(double upstreamOffset, double downstreamOffset, AMControl* lateralPV, AMControl* yawPV)
 {
-	double lateral = ( downstreamOffset + upstreamOffset ) / 2;
-	double yaw = radianToDegree(atan((downstreamOffset - upstreamOffset) / (tableHorizontalMotorPosition_ * 2)));
+	double lateral = calculateTableLateral(upstreamOffset, downstreamOffset);
+	double yaw = calculateTableYaw(upstreamOffset, downstreamOffset);
+
+	lateral = static_cast<double>(static_cast<int>(lateral*1000+0.5))/1000.0;
```

Comment:
qRound() will round to a int.

http://doc.qt.io/qt-4.8/qtglobal.html#qRound
'''
int qRound(qreal value)
Rounds value to the nearest integer.
'''
same as `qround`, qmath.h is for 4.8 and I think we are using QT4.7.3.
http://doc.qt.io/qt-4.8/qtcore-qmath-h.html

I think we are using `/usr/include/math.h`, which doesn't have a `round()`